templates/lecturers.html

```diff
@@ -0,0 +1,161 @@
+                    {{ person.user.get_full_name | question_mark_on_empty_string }}
+                  </a>
+                </td>
+                <td>
+                  {{ person.birth | question_mark_on_none_value }}
+                </td>
+                <td>
+                  {{ person.is_adult | qualified_mark }}
+                </td>
+                <td>
+                  {{ person.user.email | question_mark_on_empty_string }}
+                </td>
+                <td>
+                  {% for workshop in person.workshops %}
+                    <a href="/workshop/{{ workshop.name }}">
+                      {{ workshop.title }}
+                    </a>
+                    <br>
+                  {% endfor %}
+                </td>
+                <td>
+                  {{ person.status }}
+                </td>
+                <td>
+                  {{ person.comments }}
+                </td>
+                <td>
+                  {{ person.pesel }}
+                </td>
+                <td>
+                  {{ person.address }}
+                </td>
+                <td>
+                  {{ person.phone | question_mark_on_empty_string }}
+                </td>
+                <td>
+                  {{ person.tshirt_size }}
+                </td>
+                <td>
+                  {{ person.start_date }}
+                </td>
+                <td>
+                  {{ person.end_date }}
+                </td>
+              </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+      </div>
+    </article>
+  </div>
+{% endblock %}
+
+{% block script %}
+  <script>
+    $(document).ready(() => {
+      const table = $('#participants-table').DataTable({
+        dom: 'Bfrtip',
+        responsive: true,
+        colReorder: true,
+        buttons: [
+            {
+                extend: 'colvis',
+                columns: ':gt(0)'
+            },
+            {
+                extend: 'copy',
+                exportOptions: {
+                    columns: ':visible'
+                }
+            },
+            {
+                extend: 'excel',
+                exportOptions: {
+                    columns: ':visible'
+                }
+            },
+            {
+                extend: 'pdf',
+                exportOptions: {
+                    columns: ':visible'
+                }
+           },
+           {
+                extend: 'print',
+                exportOptions: {
+                    columns: ':visible'
+                }
+           },
+        ],
+        "columnDefs": [
+          {
+            "orderable": false,
+            targets: 0
+          },
+          {
+            visible: true,
+            targets: [0, 1, 3, 5, 6, 7]
+          },
+          {
+            visible: false,
+            targets: "_all"
+          },
+          {
+            "searchable": true,
+            targets: [1, 5, 7]
```

Comment:
Is this name, is_adult and workshops or am I reading it totally wrong? Should probably be name, phone number, address, email and workshops (double-check to match it with what participants table does)